.cron.yml

```diff
@@ -32,3 +32,9 @@ jobs:
           treeherder-symbol: bump-ac
           target-tasks-method: bump_android_components
       when: [{hour: 14, minute: 0}]
+    - name: ui-test-screenshots-x86
+      job:
+          type: decision-task
+          treeherder-symbol: ui-test-screenshots-x86-D
+          target-tasks-method: ui-test-screenshots-x86
```

Comment:
You need to a new entry in `target_tasks.py`, just like this one: 

https://github.com/mozilla-mobile/fenix/blob/4624bf5c500e54a29922a2ec56a39cc7f6db6397/taskcluster/fenix_taskgraph/target_tasks.py#L68-L75

I recommend something like: 

```py
@_target_task("screenshots")
def target_tasks_screnshots(full_task_graph, parameters, graph_config):
    """Select the set of tasks required to generate screenshots on a real devive."""

    def filter(task, parameters):
        return task.attributes.get("screenshots", False)

    return [l for l, t in full_task_graph.tasks.iteritems() if filter(t, parameters)]
```

Speaking of which, I don't think this task needs to have a long name. We may for instance want to generate screenshots on another platform. In this case, everything can happen in the same graph, which means it gets scheduled by the same decision task. So I think we can name it `screenshots`: 

```suggestion
          target-tasks-method: screenshots
```

What do you think?


By the way, you can test this change, by applying https://github.com/mozilla-mobile/shared-docs/blob/master/android/taskcluster_guide.md#how-do-i-run-a-staging-nightlyrelease-on-prs. In this case, `try_task_config.json` should look like: 

```json
{
    "parameters": {
        "optimize_target_tasks": true,
        "target_tasks_method": "screenshots"
    },
    "version": 2
}
```

Reply:
yeah! I can rename it :) only here or that name has to be consistent also in the kind.yml file too?

Reply:
No need to change `kind.yml`. `target-task-method` just applies to `target_tasks.py`